### ISSUE TITLE
🏳️‍⚧️🏳️‍🌈 Register .lgbt.sh

### ISSUE THREAD

**Issue by Franxploit** [lgbt-sh/lgbt-sh#501]
### Subdomain Name

.lgbt.sh

### DNS Record

Hacked By Franxploit

### Agreement

- [x] I have ensured that no one registered this subdomain
- [x] I have ensured that I did not reach the five subdomain limit

**Comment by Franxploit** [comment]
`<!DOCTYPE html>
<html lang="id">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="title" content="RILZZZNEWERA WAS HERE">
    <meta name="description" content="Kfstyle Team">
    <meta name="keywords" content="h4cked,deface,website,whitehat,defacer,hack,mirror, Kfstyle team">
    <meta name="robots" content="index, follow">
    <meta name="revisit-after" content="60 days">
    <meta name="author" content="Franxploit">
    <title>HACKED BY Franxploit</title>
    <meta property="og:image" content="https://files.catbox.moe/094rp1.jpg">
    <meta name="theme-color" content="#000">
    <link rel="icon" type="image/png" href="https://files.catbox.moe/094rp1.jpg">
    <link href="https://fonts.googleapis.com/css?family=Acme" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Reggae+One&amp;display=swap" rel="stylesheet">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
    
    <style type="text/css">
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }
        
        body {
            background-color: black;
            color: white;
            font-family: 'Acme', sans-serif;
            text-align: center;
            overflow: hidden;
            height:`
